### ISSUE TITLE
Support ollama

### ISSUE THREAD

**Issue by ifredom** [abi/fully-private-chat#22]

**Comment by K0-RR** [comment]
Wdym?

**Comment by abi** [comment]
I think he means having the website connect to ollama

**Comment by K0-RR** [comment]
Ohh. Is it possible without installing anything?
Because if something would need to be installed to make this connection work, then why not just install one of many front ends that are available?

**Comment by abi** [comment]
Yeah ollama would have to be installed and running. It's a good question you ask. Maybe they like this front end more than the others?
